cmd/dkvsrv/main.go

```diff
@@ -143,6 +185,10 @@ func validateFlags() {
 	if strings.ToLower(dbRole) == slaveRole && replMasterAddr == "" {
 		log.Panicf("replMasterAddr must be given in slave mode")
 	}
+	nonNullAuthFlags := btou(certPath != "") + btou(keyPath != "") + btou(caCertPath != "")
```

Comment:
For better readability + utility, consider converting `btou` function to a vararg variant that takes one or more boolean params and returns the cumulative integer count.